app/drivers/kscan/kscan_gpio_direct.c

```diff
@@ -124,6 +125,32 @@ static void kscan_direct_irq_callback_handler(const struct device *port, struct
 }
 #endif
 
+static gpio_flags_t kscan_gpio_get_extra_flags(const struct gpio_dt_spec *gpio, bool active) {
+    gpio_flags_t flags = BIT(0) & gpio->dt_flags;
```

Comment:
I don't think we need to return the first bit here at all, since that will be set in the DT still. We need to *check* it, but not include in the "extra" flags we'll use.

Minor thing, but in case we end up using this some other way later, good to really only just return the extra flags we need, not also include the active bit.